community/bkeymaps/build

```diff
@@ -1,3 +1,4 @@
 #!/bin/sh -e
 
 make DESTDIR="$1" install
+install -Dm644 -t "$1/usr/share/bkeymaps/colemak" en-latin9.bmap
```

Comment:
`-t` isn't portable iirc.

Reply:
You are correct. In fact, TIL `install` is not defined by POSIX either, and `-D` is not portable as well. All of the packaged `install` commands in KISS have the `-t` option.

Should I just use `mkdir -p` and `cp`?

Reply:
Alright. We'll leave it as-is. If we decide to make the change later we'll do it with a checklist across the entirety of the repositories. :+1: